_posts/people/2023-10-24-nataliya.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Nataliya Nechyporenko
+description: PhD Student
+layout: redirect
+redirect_to: https://nataliya.dev
+permalink: nataliya.html
+sitemap: false
+---
```

Comment:
I didn't know it was possible, and that's cool. However, if you don't have it in the sitemap, what's the purpose of having this page in the first place ?

Reply:
Also updated, the goal is to redirect my image in "people" to my website.

Reply:
Yeah sorry what I meant was that I didn't know it was possible to have a redirect just like that , but if we do we need to have it in the sitemap.

Reply:
Yep, I added sitemap: True. I believe that's the change you requested. This conversation doesn't show that change in my last push, but it's in the FilesChanged.